third_party/jetifier/jetifier-standalone/bin/jetifier-standalone

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
Couldn't you avoid this shell script stuff, and just import the jettifier standalone java and export a java_binary to execute it?  Why do work the java_binary does anyway?  What am I missing?

Reply:
crap.  didn't mean to start a review with this, just a single comment. :/

Reply:
I wonder if this was just accidentally committed, as this script is just part of https://dl.google.com/dl/android/studio/jetifier-zips/1.0.0-beta07/jetifier-standalone.zip
```
Archive:  jetifier-standalone.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  09-12-2019 02:29   jetifier-standalone/
        0  09-12-2019 02:29   jetifier-standalone/lib/
    16969  09-12-2019 02:29   jetifier-standalone/lib/jetifier-standalone.jar
   165301  09-12-2019 02:26   jetifier-standalone/lib/jetifier-processor-1.0.0-beta07.jar
    52988  09-12-2019 02:13   jetifier-standalone/lib/commons-cli-1.3.1.jar
   128890  09-12-2019 02:19   jetifier-standalone/lib/jetifier-core-1.0.0-beta07.jar
    71221  09-12-2019 02:13   jetifier-standalone/lib/asm-util-6.0.jar
    77401  09-12-2019 02:13   jetifier-standalone/lib/asm-commons-6.0.jar
    48143  09-12-2019 02:13   jetifier-standalone/lib/asm-tree-6.0.jar
    95209  09-12-2019 02:13   jetifier-standalone/lib/asm-6.0.jar
   304924  09-12-2019 02:13   jetifier-standalone/lib/jdom2-2.0.6.jar
  1290549  09-12-2019 02:13   jetifier-standalone/lib/kotlin-stdlib-1.3.41.jar
   231952  09-12-2019 02:13   jetifier-standalone/lib/gson-2.8.0.jar
   162009  09-12-2019 02:13   jetifier-standalone/lib/kotlin-stdlib-common-1.3.41.jar
    17536  09-12-2019 02:13   jetifier-standalone/lib/annotations-13.0.jar
        0  09-12-2019 02:29   jetifier-standalone/bin/
     3415  09-12-2019 02:19   jetifier-standalone/bin/jetifier-standalone.bat
     6348  09-12-2019 02:19   jetifier-standalone/bin/jetifier-standalone
---------                     -------
  2672855                     18 files

```